pkg/plugins/policies/meshtimeout/plugin/xds/configurer.go

```diff
@@ -218,3 +218,65 @@ func toProtoDurationOrDefault(d *kube_meta.Duration, defaultDuration time.Durati
 	}
 	return util_proto.Duration(d.Duration)
 }
+
+type RealResourceListenerConfigurer struct {
```

Comment:
I don't think we need to bring "RealResource" naming to this area of the code. It's kind of irrelevant from the Configurer point of view if the `api.Conf` originates from the real resource or somewhere else.

Also, there is a lot of code duplication with `ListenerConfigurer`, that'd be acceptable if we were planning to delete `ListenerConfigurer` in the future, but we can't since we need it for inbound listeners. 

What we probably should do is:
1. Mark old `ListenerConfigurer` as deprecated and use it only for legacy ToRules so we could delete it in the future. 
2. Create a new `ListenerConfigurer` (without `Rules` and `Subsets`) and use it for inbound, outbound and in the future for routes.

Reply:
you are right, changed configurer as suggested